### PR TITLE
Update to @nextcloud/vue@6.0.0-beta.4

### PR DESCRIPTION
**Pull request by juliushaertl**
- Bump to @nextcloud/vue@6.0.0-beta.3 with renaming all used components to the Nc prefix
- Update vue to 2.7.9
- Small styling fix for the controls bar

### CODE REVIEW

File: src/CardCreateDialog.vue [hunk 1]
```diff
@@ -84,17 +84,17 @@
...
-	</Modal>
+	</NcModal>
 </template>
 
 <script>
 import { generateUrl } from '@nextcloud/router'
-import Modal from '@nextcloud/vue/dist/Components/Modal'
-import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
-import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'
+import NcModal from '@nextcloud/vue/dist/Components/Modal'
```
Comment (marcelklehr) [inline src/CardCreateDialog.vue:95]: The import file probably needs changing too, right?

Comment (juliushaertl) [reply src/CardCreateDialog.vue]: Good catch, missed that in my regex replacing

File: src/components/cards/AvatarList.vue [hunk 2]
```diff
@@ -70,14 +70,14 @@
 </template>
 
 <script>
-import { Avatar, PopoverMenu, Tooltip } from '@nextcloud/vue'
+import { NcAvatar, NcPopoverMenu, Tooltip } from '@nextcloud/vue'
```
Comment (marcelklehr) [inline src/components/cards/AvatarList.vue:73]: Does Tooltip stay without prefix?

**Comment by marcelklehr** [comment]
Looks good! I found only two visual issues

Peek 2022-08-26 11-05

In the board overview:
image

File: src/components/cards/AvatarList.vue (hunk 2 above)
Comment (juliushaertl) [reply src/components/cards/AvatarList.vue:73]: Yes, only components require the two word naming enforced by eslint.

**Comment by juliushaertl** [comment]
Can't reproduce the second, the first should be fixed upstream:
Screenshot 2022-08-26 at 12 56 44

**Comment by marcelklehr** [comment]
> Can't reproduce the second

Maybe because I'm on firefox?

**Comment by juliushaertl** [comment]
Hm, same here. But I'll file issues for follow up fixes then to further investigate.

**Review by marcelklehr — APPROVED** [review]

**Comment by marcelklehr** [comment]
Cypress is also still failing

**Pull request merged**